src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -596,8 +596,6 @@ internal int GetItemsOnLastLine(int section, int itemsPerLine)
 
 		protected virtual void OnItemsSourceChanged(DependencyPropertyChangedEventArgs e)
 		{
-			Items?.Clear();
```

Comment:
We should keep this here, commented out, stating why we're not doing it here, and it does not fail a test. We should probably be adding a simple one with the existing.

Reply:
Done